src/test/java/com/google/step/youtube/YouTubeInfoScraperTest.java

```diff
@@ -58,6 +62,8 @@
     private static final String UPLOAD_ID = "UPLOAD_ID";
     private static final String VIDEO_ID = "VIDEO_ID";
     private static final String VIDEO_TITLE = "VIDEO_TITLE";
+    private static final String KEYWORD = "KEYWORD";
+    private static final String KEYWORD_NO_RESULTS = "KEYWORD_NO_RESULTS";
```

Comment:
Change to NO_RESULTS_KEYWORD to stay consistent with putting adjective before noun.